client/src/js/directives/payslip_employee.js

```diff
@@ -0,0 +1,19 @@
+/**
+  * This Directive is responsible for generate an employee block of data
+  */
+angular.module('bhima.directives')
+.directive('bhimaPayslipEmployee', function () {
+  return {
+    restrict    : 'E',
+    templateUrl : '/partials/templates/payslipEmployee.tmpl.html',
+    scope       : {
+      row             : '=employee',
```

Comment:
Why not call this `employee`?  It would be something like `scope : { employee : '=' }`.